assets/js/base/components/country-input/style.scss

```diff
@@ -0,0 +1,5 @@
+.wc-block-components-country-input,
+.wc-block-components-state-input {
```

Comment:
I'm not sure it's a good idea to have `state-input` styles in the `country-input` folder. Ideally each component should have its own styles in its folder. I would either:
1. duplicate this ruleset in `base/components/state-input/style.scss` or
2. move these styles to `base/components/select/style.scss`

How does it sound?

Reply:
I wanted to avoid `base/components/select/style.scss` because that's a child component, will create a state input style.